system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -34,7 +35,10 @@ class TopSite extends React.Component {
               <span className="letter-fallback">{title[0]}</span>
               <div className={screenshotClassName} style={style} />
           </div>
-          <div className="title">{title}</div>
+          <div className={classNames("title", {pinned: link.isPinned})}>
```

Comment:
since we don't have a license to use ```classnames``` in mozilla central, can you just do class names more like: https://github.com/mozilla/activity-stream/blob/master/system-addon/content-src/components/TopSites/TopSites.jsx#L30 ? (and also change it in the prefs pane file 😳